redskyctl/internal/commands/generate/rbac.go

```diff
@@ -90,9 +91,21 @@ func NewRBACCommand(o *RBACOptions) *cobra.Command {
 func (o *RBACOptions) Complete() {
 	// Create a REST mapper to convert from GroupVersionKind (used on patch targets) to GroupVersionResource (used in policy rules)
 	rm := meta.NewDefaultRESTMapper(scheme.Scheme.PreferredVersionAllGroups())
+
+	// Add all types known to the Kubernetes Client Go scheme (we can use unsafe guess safely for these types)
 	for gvk := range scheme.Scheme.AllKnownTypes() {
 		rm.Add(gvk, meta.RESTScopeRoot)
 	}
+
+	// If we hit a CRD we will use "unsafe guess" (basically add an "s"); here are some well-known exceptions
```

Comment:
Should we shell out to kubectl and fill in the info?
```
kubectl get crd experiments.redskyops.dev -o go-template='{{ .spec.names.plural }}'
kubectl get crd experiments.redskyops.dev -o go-template='{{ .spec.names.singular }}'
```

Reply:
I like it.